src/pages/distroSettings/tabs/GeneralTab/getFormSchema.ts

```diff
@@ -88,6 +93,10 @@ export const getFormSchema = (
     },
     distroOptions: {
       "ui:ObjectFieldTemplate": CardFieldTemplate,
+      adminOnly: {
+        "ui:description":
+          "Admin-only distros are not selectable by general users (e.g. when spawning a host). They do not have their access controlled beyond being hidden.",
```

Comment:
the example `"when spawning a host"` is somewhat confusing to me because I think we already hide distros in the spawn host modal through the `userSpawnAllowed` attribute. Unless maybe `adminOnly` is going to take precedence over `userSpawnAllowed` in the next PR? 👀 otherwise LGTM!

Reply:
Thomas clarified that there are lots of these temporary distros where `userSpawnEnabled` is true. The next PR will have admin-only take precedence over that setting, but will still allow these distros to be visible to admins in the dropdown.